src/utils.py

```diff
@@ -0,0 +1,47 @@
+def _sort_step_order(input_steps):
```

Comment:
What is this for?

Reply:
I just assumed user could enter the steps in arbitrary order like "eval" "ft" "deploy"... This sort them out in the right order "ft" "eval" "deploy" ...